data-prepper-plugins/geoip-processor/README.md

```diff
@@ -68,27 +68,11 @@ pipeline:
 
 ## Sample JSON input:
 
-"peer" : {
-"ip" : "1.2.3.4"
-"host" : "example.org"
-}
-"status" : "success"
+{"peer": { "ips":{ "src_ip1" : "8.8.8.8", "dst_ip1": "8.8.8.9" }, "host": "example.org" }, "status": "success"}
 
 ## Sample JSON Output:
 
-"peer" : {
-"ip" : "1.2.3.4"
-"host" : "example.org"
-}
-"location" : {
-"status" : "success"
-"country" : "United States"
-"city_name" : "Seattle"
-"latitude" : "47.64097"
-"longitude" : "122.25894"
-"zip_code" : "98115"
-}
-
+{"peer":{"ips":{"src_ip":"8.8.8.8","dst_ip":"8.8.8.9"},"host":"example.org"},"status":"success","target1":{"continent_name":"North America","country_iso_code":"US","timezone":"America/Chicago","ip":"8.8.8.8","country_name":"United States","location":{"lon":-97.822,"lat":37.751},"organization_name":"GOOGLE","asn":15169,"network":"8.8.8.0/24"},"target2":{"continent_name":"North America","country_iso_code":"US","timezone":"America/Chicago","ip":"8.8.8.9","country_name":"United States","location":{"lon":-97.822,"lat":37.751},"organization_name":"GOOGLE","asn":15169,"network":"8.8.8.0/24"}}
```

Comment:
Seems like formatting got lost on this line

Reply:
Formatted now with proper JSON formatter. Please check now.